frontend/src/employee-frontend/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -428,8 +428,8 @@ export default React.memo(function VoucherServiceProviderUnit() {
                         <FixedSpaceColumn spacing="xs">
                           <Link to={`/child-information/${row.childId}`}>
                             {formatName(
-                              row.childFirstName,
                               row.childLastName,
+                              row.childFirstName,
                               i18n
                             )}
```

Comment:
This does the trick, but `formatName` also has an optional 4th parameter `lastNameFirst` 😄